awsshell/utils.py

```diff
@@ -123,12 +125,19 @@ def file_exists(self, filename):
 
 def force_unicode(obj, encoding='utf8'):
     """Recursively search lists and dicts for strings to encode as unicode."""
+    def attempt_decode(string):
```

Comment:
Good catch, I think generally this isn't a super big deal but since this is a recursive function the redefinition could be pretty expensive.